src/icons/icons.qrc

```diff
@@ -249,6 +249,7 @@
         <file>flags/za.svg</file>
         <file>flags/zm.svg</file>
         <file>flags/zw.svg</file>
+        <file>icons.json</file>
```

Comment:
I would like more specific name, e.g. "iconmapping.json" or "iconconfig.json"

Reply:
@jagannatharjun:  IMO, `iconmapping.json` sounds more descriptive.

Reply:
Or just `config.json` if it is supposed to be in `icons` directory.